### DISCUSSION TITLE
Does `access_mode` apply to `postgres_attach`?

### DISCUSSION THREAD

**Discussion by siegerts** [duckdb/duckdb#5702] [Q&A] [answered]
Just wondering what access is granted through `postgres_attach`. I'm using the Node API and would like to restrict the scanner to only allow reads.

**Comment by Alex-Monahan** [comment]
Howdy! I believe that you can adjust that on the Postgres side. Just grant the user you connect with the appropriate permissions in Postgres.

**Comment by Mytherin** [comment, marked as answer]
postgres_attach currently only allows reading tables -  so it should always be read-only.

**Reply by siegerts** [reply]
Thanks! That's good to know!
